rocAL/rocAL/include/readers/video/video_reader.h

```diff
@@ -99,7 +43,7 @@ class VideoReader
     /*!
      \param desc  User provided descriptor containing the files' path.
     */
-    virtual Status initialize(VideoReaderConfig desc) = 0;
+    virtual Status initialize(ReaderConfig desc) = 0;
 
     //! Reads the next resource item
     virtual SequenceInfo get_sequence_info() = 0;
```

Comment:
I see the SequenceInfo definition is removed and used here

Reply:
Moved back to video_reader.h